### PR TITLE
Added in Socket connections to login/signup

### PR DESCRIPTION
**Pull request by jamesramadan**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+module.exports = {
```
Comment (hannahhenderson) [inline .eslintrc.js]: Please remove this file

File: src/login/login.js [hunk 2]
```diff
@@ -15,6 +15,20 @@ export default class Login extends Component {
...
+  validateUser() {
+    this.props.socket.emit('validateUserLogin', { 
+        username: this.state.username,
+        password: this.state.password
+    });
+  }
+
+  componentWillMount() {
+    this.props.socket.on('Authentication', username => {
+      console.log('heyo2');
```
Comment (hannahhenderson) [inline src/login/login.js]: Check the 'log' function and wrapper on Maps.js to get around eslint issues with console.log

File: src/login/login.js [hunk 3]
```diff
@@ -33,38 +47,38 @@ export default class Login extends Component {
...
-                    <Text style={styles.label}>LOGIN</Text>
-                </TouchableHighlight>
-                <Hyperlink>
-                            <View>
-                                <Text style={{fontSize:15}}>
-                                Not a member? <Text onPress={() => this.onLinkPress()} style={{color: 'blue', fontSize:15}}>Create a Profile</Text>
-                                </Text>
-                            </View>
-                </Hyperlink>
+          <Image source={require("./cookie.jpg")} />
```
Comment (hannahhenderson) [inline src/login/login.js:95]: change to import

File: src/login/login.js [hunk 4]
```diff
@@ -77,12 +91,12 @@ const styles = StyleSheet.create({
     flex: 1,
     justifyContent: 'center',
     alignItems: 'center',
-    backgroundColor: '#87CEEB',
+    backgroundColor: '#87CEEB'
```
Comment (hannahhenderson) [inline src/login/login.js]: Add trailing commas back in this section

**Comment by hannahhenderson** [comment]
Other than comments, lgtm

File: .eslintrc.js (hunk 1 above)
Comment (jamesramadan) [inline .eslintrc.js]: removed

File: src/login/login.js (hunk 2 above)
Comment (jamesramadan) [inline src/login/login.js]: removed console logs

**Pull request merged**